python/ray/_private/runtime_env/packaging.py

```diff
@@ -451,6 +487,38 @@ def get_uri_for_package(package: Path) -> str:
         )
 
 
+def get_uri_for_file(file: str) -> str:
+    """Get a content-addressable URI from a file's content.
+
+    This function will generate the name of the package by the file.
+    The final package name is: _ray_pkg_<HASH_VAL>.zip of this package.
```

Comment:
```suggestion
    The final package name is _ray_pkg_<HASH_VAL>.zip, where <HASH_VAL> is a hash of this package.
```

Reply:
I couldn't understand what you meant to say in this sentence. This is a guess at a rewrite. Please correct if needed.